atat_internal_api.yaml

```
@@ -0,0 +1,2197 @@
+        '404':
+          description: Task Order with the given ID does not exist or is pending processing
+      x-amazon-apigateway-request-validator: "request-params-validator"
+      x-amazon-apigateway-integration:
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: "arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${NotImplementedFunction.Arn}/invocations"
+        type: "aws_proxy"
+      security:
+        Fn::If:
+          - IsAuthorizationRequired
+          -
+            - cognitoAuthorizer: []
+          - Ref: AWS::NoValue
+    options:
+      tags:
+        - cors
+      description: CORS headers
+      responses:
+        '200':
+          $ref: '#/components/responses/CorsHeaders'
+      x-amazon-apigateway-integration:
+        type: mock
+        requestTemplates:
+          application/json: |
+            {
+              "statusCode" : 200
+            }
+        responses:
+          default:
+            statusCode: "200"
+            responseParameters:
+              method.response.header.Access-Control-Allow-Headers: '''*'''
+              method.response.header.Access-Control-Allow-Methods: '''*'''
+              method.response.header.Access-Control-Allow-Origin: '''*'''
+            responseTemplates:
+              application/json: "{}"
+components:
+  parameters:
+    offsetParam:
+      in: query
+      name: offset
+      required: false
+      schema:
+        type: integer
+        minimum: 0
+      description: The number of items to skip before starting to collect the result set.
+    limitParam:
+      in: query
+      name: limit
+      required: false
+      schema:
+        type: integer
+        minimum: 1
+        maximum: 50
+        default: 20
+      description: The numbers of items to return.
+  responses:
+    CorsHeaders:
+      description: Default response for CORS method
+      headers:
+        Access-Control-Allow-Origin:
+          schema:
+            type: string
+        Access-Control-Allow-Methods:
+          schema:
+            type: string
+        Access-Control-Allow-Headers:
+          schema:
+            type: string
+      content: {}
+  schemas:
+    BaseObject:
+      type: object
+      properties:
+        updatedAt:
+          type: string
+          format: "date-time"
+          readOnly: true
+        createdAt:
+          type: string
+          format: "date-time"
+          readOnly: true
+        archivedAt:
+          type: string
+          format: "date-time"
+          readOnly: true
+        id:
+          type: string
+          readOnly: true
+      additionalProperties: false
+      description: "Base model common to all first-class objects (those with IDs)"
+    Error:
+      type: object
+      additionalProperties: false
+      description: Generic error model
+      properties:
+        code:
+          type: string
+          enum:
+          - INVALID_INPUT
+          - OTHER
```

```suggestion
          enum:
            - INVALID_INPUT
            - OTHER
```

Minor formatting thing in a few places. Can we be consistent and indent list items?